src/Components/LoginPage.tsx

```diff
@@ -26,29 +30,31 @@ export function LoginPage() {
         //CF 2022-10-16
         //Fetch is asynchronous, so it returns a Promise.  When it is resolved (the request is completed),
         // it moves onto the then block. If an error is thrown, it is caught in the catch block.
-        fetch(process.env.REACT_APP_API_BASE + '/users/login', {
+        fetch(env.REACT_APP_API_BASE + '/users/login', {
             method: 'POST',
             body: JSON.stringify(data),
             headers: {
                 'Content-Type': 'application/json'
             },
             credentials: 'include'
-        }).then((response) => {
+        }).then( async (response) => {
             //response.json() returns a promise
-            response.json().then((body) => {
+            const body:Login = await response.json()
 
-                if(!response.ok) {
-                    setAlertMessage(body.data.message);
-                    setAlertVisible(true);
-                    return;
-                }
-
-                setLoading(false);
-                navigate(`/`);
-            });
+            if(!response.ok) {
+                setAlertMessage(body.data.message);
+                setAlertVisible(true);
+                return;
+            }
+            const user:User = await fetch(env.REACT_APP_API_BASE + '/users/' + body.data.user_id, {credentials: 'include'}).then(b => b.json())
```

Comment:
User cookie is already being created in UserLoginAction.php and UserPostAction.php. I believe it is more efficient to do it there so you do not have to wait for an additional fetch to return the user.

Reply:
I agree.  We could return the information about the user in the initial response to avoid the additional request.

Reply:
I removed the cookie from the client side

https://github.com/thebaker10/Fresh-Mart/pull/44/commits/09b6dce1fd9e5a170d6c9b4654304b7aa2dbc75d